conf/machine/imx7d-remarkable2.conf

```diff
@@ -28,9 +28,17 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += " \
     u-boot-remarkable2 \
 "
 
-MACHINE_EXTRA_RRECOMMENDS += " \
-  linux-firmware-bcm43456 \
+# We mark all the firmware as MACHINE_ESSENTIAL_EXTRA instead of MACHINE_EXTRA
+# as the rM2 is a consumer product and users expect things like WiFi,
+# touchscreen ect to work even on minimal images. Also without WiFi there is
+# no way to interact via the command line with the system (UART RX doesn't
+# work.)
+# We still make this as RRECOMMENDS instead of RDEPENDS as it isn't
+# required for boot.
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
   linux-firmware-imx-sdma-imx7d \
+  linux-firmware-bcm43455 \
+  linux-firmware-bcm43456 \
```

Comment:
This is conceptually wrong. This has nothing to do with expectations but machine working or not. WiFi is nice to have but is not essential for it to work. 

The "Also without WiFi there is no way to interact via the command line with the system (UART RX doesn't work.)" argument is the only sensible argument to raise its importance.

Reply:
Ok, I have revered this change.